index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="container">
+      <header class="toolbar">
+        <img src="images/image_geometry_1-1.png" alt="logo" class="shapes" />
+        <img src="images/ic_menu.png" alt="menu-bar" class="menu" />
```

Comment:
- [ ] Please while you did great work here 👍️, I would kindly suggest you again put your hamburger menu icon inside a `nav` tag and wrap it with an `a` tag. This is because of semantic reasons. [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.
```
 <nav>
     <a><img src="images/ic_menu.png" alt="menu-bar" class="menu" /></a>
 </nav>
```